app/src/main/java/com/mashup/mobalmobal/data/repository/SignRepository.kt

```diff
@@ -9,10 +10,13 @@ import io.reactivex.Single
 import javax.inject.Inject
 
 class SignRepository @Inject constructor(private val service: SignService) {
-
-    fun login(): Single<Response<UserDto>> {
-        return Single.just(Response())
-    }
+    fun login(
+        fireStoreId: String
+    ): Single<Response<LoginDto>> = service.login(
```

Comment:
```suggestion
    fun login(fireStoreId: String): Single<Response<LoginDto>> = 
        service.login(
```
필수는 아닌데 한 줄에 쓸 수 있으면 최대한 한 줄에 쓰는게 좋을 것 같아요